src/test/acl.test.ts

```diff
@@ -196,7 +196,6 @@ describe('ACL', () => {
         const allowedBook = permission.filter(book);
 
         expect(permission.granted).toBe(true);
-        expect(permission.attributes).toEqual(['*']);
```

Comment:
Why is this removed?

Reply:
I added `expect(permission.attributes.sort()).toEqual(['*', 'author', 'title']);`

The part `attributes: !attributes.includes('*') ? attributes : ['*'],` was changed. This is why `attributes` contains `author` and `title`. There should be changed the attributes filter - proposal. Should I create the issue?

Reply:
Please, do.

Reply:
:heavy_check_mark: issue #8 created